pwrsupply/pstypes-setpoint-limits.txt

```diff
@@ -86,15 +86,23 @@ ts-corrector-cv-strong-hlim  -0012.0   -0012.0   -0012.0   -0012.0   +0012.0   +
-si-quadrupole-q14-100A-fam   +0000.0   +0000.0   +0000.0   +0000.0   +0160.0   +0160.0   +0160.0   +0160.0   0.1       0.20      +0032.0   +0000.1
-si-quadrupole-q20-160A-fam   +0000.0   +0000.0   +0000.0   +0000.0   +0160.0   +0160.0   +0160.0   +0160.0   0.1       0.20      +0032.0   +0000.1
-si-quadrupole-q20-150A-fam   +0000.0   +0000.0   +0000.0   +0000.0   +0150.0   +0150.0   +0150.0   +0150.0   0.1       0.20      +0030.0   +0000.1
-si-quadrupole-q20-140A-fam   +0000.0   +0000.0   +0000.0   +0000.0   +0160.0   +0160.0   +0160.0   +0160.0   0.1       0.20      +0032.0   +0000.1
-si-quadrupole-q20-120A-fam   +0000.0   +0000.0   +0000.0   +0000.0   +0120.0   +0120.0   +0120.0   +0120.0   0.1       0.20      +0024.0   +0000.1
-si-quadrupole-q30-150A-fam   +0000.0   +0000.0   +0000.0   +0000.0   +0160.0   +0160.0   +0160.0   +0160.0   0.1       0.20      +0032.0   +0000.1
+#si-quadrupole-q14-150A-fam   +0000.0   +0000.0   +0000.0   +0000.0   +0160.0   +0160.0   +0160.0   +0160.0   0.1       0.20      +0032.0   +0000.1
+#si-quadrupole-q14-100A-fam   +0000.0   +0000.0   +0000.0   +0000.0   +0160.0   +0160.0   +0160.0   +0160.0   0.1       0.20      +0032.0   +0000.1
+si-quadrupole-q14-qdb2-fam   +0000.0   +0000.0   +0000.0   +0000.0   +0160.0   +0160.0   +0160.0   +0160.0   0.1       0.20      +0032.0   +0000.1
+si-quadrupole-q14-qdp2-fam   +0000.0   +0000.0   +0000.0   +0000.0   +0160.0   +0160.0   +0160.0   +0160.0   0.1       0.20      +0032.0   +0000.1
+si-quadrupole-q14-qda-fam    +0000.0   +0000.0   +0000.0   +0000.0   +0160.0   +0160.0   +0160.0   +0160.0   0.1       0.20      +0032.0   +0000.1
+si-quadrupole-q14-qdb1-fam   +0000.0   +0000.0   +0000.0   +0000.0   +0160.0   +0160.0   +0160.0   +0160.0   0.1       0.20      +0032.0   +0000.1
+si-quadrupole-q14-qdp1-fam   +0000.0   +0000.0   +0000.0   +0000.0   +0160.0   +0160.0   +0160.0   +0160.0   0.1       0.20      +0032.0   +0000.1
+si-quadrupole-q20-q2-fam     +0000.0   +0000.0   +0000.0   +0000.0   +0160.0   +0160.0   +0160.0   +0160.0   0.1       0.20      +0032.0   +0000.1
+si-quadrupole-q20-q4-fam     +0000.0   +0000.0   +0000.0   +0000.0   +0150.0   +0150.0   +0150.0   +0150.0   0.1       0.20      +0030.0   +0000.1
+si-quadrupole-q20-qfa-fam    +0000.0   +0000.0   +0000.0   +0000.0   +0160.0   +0160.0   +0160.0   +0160.0   0.1       0.20      +0032.0   +0000.1
+si-quadrupole-q20-q1-fam     +0000.0   +0000.0   +0000.0   +0000.0   +0120.0   +0120.0   +0120.0   +0120.0   0.1       0.20      +0024.0   +0000.1
+si-quadrupole-q20-q3-fam     +0000.0   +0000.0   +0000.0   +0000.0   +0120.0   +0120.0   +0120.0   +0120.0   0.1       0.20      +0024.0   +0000.1
+si-quadrupole-q30-fbb-fam    +0000.0   +0000.0   +0000.0   +0000.0   +0160.0   +0160.0   +0160.0   +0160.0   0.1       0.20      +0032.0   +0000.1
+si-quadrupole-q30-fbp-fam    +0000.0   +0000.0   +0000.0   +0000.0   +0160.0   +0160.0   +0160.0   +0160.0   0.1       0.20      +0032.0   +0000.1
```

Comment:
wouldn't it be `si-quadrupole-q30-qfb-fam` and `si-quadrupole-q30-qfp-fam`?

Reply:
yes! thanks.

Reply:
fixed, @anacso17

Reply:
missing a push, maybe?